src/gba/cart/vfame.c

```diff
@@ -65,13 +76,18 @@ void GBAVFameDetect(struct GBAVFameCart* cart, uint32_t* rom, size_t romSize) {
 		mLOG(GBA_MEM, INFO, "Vast Fame game detected");
 	}
 
-	// This game additionally operates with a different set of SRAM modes
-	// Its initialisation seems to be identical so the difference must be in the cart HW itself
+	// These games additionally operates with a different set of SRAM modes
+	// Their initialisation seems to be identical so the difference must be in the cart HW itself
 	// Other undumped games may have similar differences
 	if (memcmp("George Sango", &((struct GBACartridge*) rom)->title, 12) == 0) {
 		cart->cartType = VFAME_GEORGE;
 		mLOG(GBA_MEM, INFO, "George mode");
 	}
+	// Chinese version of Digimon Sapphire; header is identical to the English version which uses the normal reordering
+	// so we have to use some other way to detect it
+	else if (doCrc32(rom, romSize) == 0x793A328F) {
+		cart->cartType = VFAME_STANDARD;
```

Comment:
Did you mean to assign ALTERNATE here? Otherwise, it's never assigned.

Reply:
oops, my bad there, yeah I did